admin/listPurchases/listView.html

```diff
@@ -279,7 +279,8 @@
     dispatchButton.classList.add("is-clicked");
     requestMarkOrderDispatched(id, (error) => {
         if(error) {
-            document.write(error.response);
+            alert("couldn't mark order dispatched");
```

Comment:
To make this case more actionable we should probably also use `console.error` to log out the actual error object, so we can debug what went wrong by using the browser console